rs_Introduction_to_Python.md

```diff
@@ -0,0 +1,149 @@
+
+# Introduction to Using Python for Image Analysis
+## Setting up your Enviroment
+To get started using python the first step is the installation and there are several ways you can do it. To make it easier for your future self it's a good idea to set up a virtual environment. You can think of this as a virtual installation folder where you install all your python libraries (basically just functions that other people have written).
+One way to manage these environments is [anaconda](https://www.anaconda.com/products/individual)
+This will not only install anaconda, the software to manage your environments but also a lot of libraries commonly needed for scientific python programs. 
+
+Once installed you can manage your environments via the command line. Every operating system has one and they differ slightly.
+I'm using Windows and to open it you just need to search for 'cmd' using the Windows search close to the Start menu and will find it.
+The basic functionality and use of conda in it can be found in this [great explanation of command line and conda](https://youtu.be/MOEPe9TGBK0?t=1161), created by Robert Haase, my supervisor.
+Another nice explanation of the command line can be found by Sreeni [here](https://www.youtube.com/watch?v=AzuajQnRuGI&list=PLZsOBAyNTZwbIjGnolFydAN33gyyGP7lT&index=5&ab_channel=DigitalSreeni) 
+
+You will also need an integrated development environment (in short IDE), which makes organizing code, files and projects easier. You can watch more details and explanations on why IDEs make sense [here](https://www.youtube.com/watch?v=BxebisJKWBg&list=PLZsOBAyNTZwbIjGnolFydAN33gyyGP7lT&index=7&ab_channel=DigitalSreeniDigitalSreeni). If you go with anaconda, like I did, one IDE will be preinstalled in your environment: [spyder](https://www.spyder-ide.org/) (it's the same as the one used by Sreeni).
+
+To get started with your first programs in python, I would suggest you follow the tutorial on setting up an environment after installing anaconda. I can offer a bonus tip for creating conda environments and that is to install a specific python version, as some libraries won't work with the newest python version but you may find that out yourself. Furthermore, install all the essential scientific libraries included in anaconda. To do this you need to initialise a conda environment as follows:
+
+```sh
+C:\Users\yourusername> conda activate
+C:\Users\yourusername> conda create -n your_environment_name python=3.8 anaconda 
```

Comment:
Hi Ryan,

do you maybe also want to show how to activate your newly created environment?